### DISCUSSION TITLE
recorded sound is not clear with a crackle sound

### DISCUSSION THREAD

**Discussion by peabody1234** [ExistentialAudio/BlackHole#629] [General]
When recording with blackhole will there always be a crackle sound on a MAC?

**Comment by devinroth** [comment]
No. Probably because you don’t have drift correction enabled.

Devin Roth

**Comment by jamesaustin** [comment]
I've been trying to use BlackHole with `ffmpeg` and could never create a recording without continuous crackling on an M1 MacBook Pro 12.4.

I switched to `sox` and it worked perfectly first time.

Just FYI incase anyone else is hitting this.

**Reply by KingAab001** [reply]
I have the same problem, how did you switch from ffmpeg to SoX?

**Reply by jamesaustin** [reply]
`sox -t coreaudio "BlackHole 2ch" output.wav`

Will capture the BlackHole device to a file.
